Source/Core/Core/HW/EXI/EXI_DeviceIPL.cpp

```diff
@@ -102,7 +104,8 @@ CEXIIPL::CEXIIPL()
 
   // Load whole ROM dump
   // Note: The Wii doesn't have a copy of the IPL, only fonts.
-  if (!SConfig::GetInstance().bWii && LoadFileToIPL(SConfig::GetInstance().m_strBootROM, 0))
+  if (!SConfig::GetInstance().bWii && Config::Get(Config::MAIN_LOAD_IPL_FONTS) &&
```

Comment:
Since you're using this to control not just whether fonts are loaded from the IPL but also whether the whole IPL is loaded into RAM, it would be better to have the name contain just IPL where you currently have IPL_FONTS.

Reply:
Sure, that sounds good to me.